broker/src/tunneldigger_broker/network.py

```diff
@@ -35,8 +32,9 @@ def __init__(self, address, interface):
         """
 
         self.socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+        self.socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEPORT, 1)
```

Comment:
Could you add a comment explaining why this is needed?

Reply:
Done.